.github/workflows/ci.yml

```diff
@@ -17,7 +17,18 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Build & Run cargo tests
-        run: RUSTFLAGS="-D warnings" cargo test --all-features --workspace
+        run: |
```

Comment:
Could you put all features to list, iterat over that list and execute the same command, please?=)